src/Lib/Managers/NobManager.vala

```diff
@@ -66,7 +66,13 @@ public class Akira.Lib.Managers.NobManager : Object {
     private double height_offset_y;
     private double bb_width;
     private double bb_height;
-    Cairo.Matrix bb_matrix;
+    private Cairo.Matrix bb_matrix;
+
+    // Values for the Transform Panel fields.
+    public double selected_x;
+    public double selected_y;
+    public double selected_width;
```

Comment:
what is the difference between bbwidth / selected_width?

Reply:
For now there's no difference at all, but I created that because it's specific to the Transform Panel and I don't know if the bbwidth will remain as it is.
I can definitely drop it if you think it's redundant and you don't think the bbwidth and bbheight will drastically change their purpose (who knows really, tho).
I can write some comments to be sure those variables are referencing the transform panel.